### PR TITLE
Feature/enums

### PR DESCRIPTION
**Pull request by paul-burnett**

### CODE REVIEW

File: validationFunctions/validateBusinessType.js [hunk 1]
```diff
@@ -20,13 +20,13 @@ const validateBusinessType = type => {
...
 
-    const trimmedValidBusinessTypes = validBusinessTypes.map(validType =>
-      trim(validType)
-    );
-
-    const matchesValidBusinessTypesList = trimmedValidBusinessTypes.includes(
-      trim(type)
-    );
+    let matchesValidBusinessTypesList = false;
+    Object.keys(businessTypeEnum).forEach(validType => {
```
Comment (simon-says-test) [inline validationFunctions/validateBusinessType.js:24]: is this the same as doing `return businessTypeEnum[trim(type)] ? true : false` ?

File: package.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "@slice-and-dice/register-a-food-business-validation",
-  "version": "1.18.0",
+  "version": "1.19.0",
   "description": "Validation functions for the register a food business service",
   "main": "index.js",
   "scripts": {
-    "format:verify": "prettier --list-different validationFunctions/**/*.js ./valid-business-types.json",
-    "format": "prettier --write validationFunctions/**/*.js ./valid-business-types.json",
-    "lint": "eslint ./validationFunctions",
```
Comment (simon-says-test) [inline package.json:9]: it formats these horribly does it?

File: update-business-types.js [hunk 3]
```diff
@@ -3,38 +3,48 @@ const fetch = require("node-fetch");
...
+    }
+    return displayNames;
+  }
 
   newBusinessTypesArray.forEach(businessType => {
-    businessType.searchTerms.forEach(searchTerm => {
-      const newArrayEntry = businessType.displayName;
-      transformedBusinessTypeArray.push(newArrayEntry);
-    });
+    const displayNames = getDisplayNames(businessType["rdfs:label"]);
```
[the remark this answers is not in the document]
Comment (paul-burnett) [reply update-business-types.js:30]: rdfs:label is always the same as skos:prefLabel at the moment, but have updated it to use prefLabel anyway.

File: validationFunctions/validateBusinessType.js (hunk 1 above)
Comment (paul-burnett) [reply validationFunctions/validateBusinessType.js:24]: Yes, will update to match the others enum validators

File: package.json (hunk 2 above)
Comment (paul-burnett) [reply package.json:9]: valid-business-types.json no longer exists, moved into enums folder.

Comment (simon-says-test) [reply package.json:9]: sorry yeah - I can't read.

**Review by simon-says-test — APPROVED** [review]
Comments addressed - good to go

**Pull request merged**